tests/integration/cloudformation/test_cloudformation_acm.py

```diff
@@ -0,0 +1,25 @@
+from localstack.utils.common import short_uid
+
+TEST_TEMPLATE = """
+Resources:
+  cert1:
+    Type: "AWS::CertificateManager::Certificate"
+    Properties:
+      DomainName: "{{domain}}"
+      DomainValidationOptions:
+        - DomainName: "{{domain}}"
+          HostedZoneId: zone123  # using dummy ID for now
+      ValidationMethod: DNS
+"""
+
+
+def test_cfn_apigateway_aws_integration(
```

Comment:
I guess this is a case where running against AWS won't make sense right? In that case I'd suggest adding the `@only_localstack` annotation here to make this explicit